Source: Timo Gurr <tgurr@exherbo.org>
Upstream: no
Reason: Remove unneeded SUSE legacy init compatibility stuff resulting in a warning in the journal

diff -Naur mdadm-3.3.2.old/systemd/mdmonitor.service mdadm-3.3.2/systemd/mdmonitor.service
--- mdadm-3.3.2.old/systemd/mdmonitor.service	2014-08-21 12:16:56.000000000 +0200
+++ mdadm-3.3.2/systemd/mdmonitor.service	2014-08-22 12:46:16.164612967 +0200
@@ -11,6 +11,4 @@
 
 [Service]
 Environment=  MDADM_MONITOR_ARGS=--scan
-EnvironmentFile=-/run/sysconfig/mdadm
-ExecStartPre=-/usr/lib/mdadm/mdadm_env.sh
 ExecStart=BINDIR/mdadm --monitor $MDADM_MONITOR_ARGS
